src/wizard/rust/profiling-onboarding/rust/1.install.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Rust
+doc_link: https://docs.sentry.io/platforms/rust/profiling/
+support_level: alpha
+type: language
+---
+
+#### Install
+
+For the Profiling integration to work, you must have the Sentry Rust SDK crate (minimum version 0.29.0). Learn more about installation methods in our [full documentation](https://docs.sentry.io/platforms/rust/#install).
+
+```cargo
+[dependencies]
+sentry = { version = "0.29.0", features = ["profiling"] }
```

Comment:
I believe we have some magic snippet that automatically fills in the latest version. Other examples / wizards should have that.

Reply:
nice, let me fix this then! :)